src/components/mortgage-investment-compare/input-card.js

```diff
@@ -16,18 +17,57 @@ import {
 import { toUSD } from '../../utils/numberFormat'
 
 /**
+ *
  * @param {*} props
- *   @property {number} loanAmt
- *   @property {number} investmentRate
- *   @property {number} inflation
- *   @property {function} onInputChange
- *   @property {Object} option1
- *   @property {Object} option2
- *   @property {Object} shorterOption
- *   @property {Object} longerOption
+ *   @property {number} mustBeGreaterThan - If present, number must be greater
  */
+function ValidatedNumberInput (props) {
```

Comment:
I love this. Hopefully we can use it for other tools in the future. I think we should move it to a different file and import it accordingly like we do with the accordion